pebble.go

```diff
@@ -0,0 +1,525 @@
+package db
+
+import (
+	"bytes"
+	"fmt"
+	"path/filepath"
+	"runtime"
+	"sync"
+
+	"github.com/cockroachdb/pebble"
+	"github.com/cockroachdb/pebble/bloom"
+)
+
+// ForceSync
```

Comment:
Have we established whether this version of Pebble needs these extra steps?

Reply:
It needs them, as long as SDK doesn't cleanly. close the DB after an upgrade -- at least I think that's what @baabeetaa found.